### PR TITLE
Readme.md - updated for Samsara

### PR DESCRIPTION
**Pull request by andrewpalumbo**
Updated readme.md from #130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+To use the Samsara environment you'll need to include both the engine neutral math-scala dependency:
+```
+<dependency>
+    <groupId>org.apache.mahout</groupId>
+    <artifactId>mahout-math-scala_2.10</artifactId>
+    <version>${mahout.version}</version>
+</dependency>
+```
+and a backing engine, e.g:
```
Comment (dlyubimov) [inline README.md]: is the "backing engine" a correct euphemism for this?
"backend engine" maybe?

**Comment by dlyubimov** [comment]
+1 overall.

File: README.md (hunk 1 above)
Comment (dlyubimov) [inline README.md]: also we are not adding "engine", we are adding a translation for a specific backend engine. (i.e. we are not adding "spark", we are adding a translation for A. Spark)

Comment (andrewpalumbo) [inline README.md]: how about "and a dependency for back end engine translation, e.g:" ?

Comment (dlyubimov) [inline README.md]: +1

Comment (andrewpalumbo) [inline README.md]: done

**Pull request closed without merging**